internal/provider/data_source_index.go

```diff
@@ -90,7 +90,7 @@ func dataSourceIndex() *schema.Resource {
 							Elem:        &schema.Schema{Type: schema.TypeString},
 							Set:         schema.HashString,
 							Computed:    true,
-							Description: "List of replica names.",
+							Description: "List of replica names. Names of virtual replicas should be surrounded with `virtual()`",
```

Comment:
:thinking: This does not seem to have an effect… oh I changed the wrong file, didn't I?

Reply:
Done :)